### ISSUE TITLE
Backslash character "\" gets automatically escaped

### ISSUE THREAD

**Issue by kojimaru14** [box/boxcli#485]
### I have checked that the SDK documentation doesn't solve my issue.

- [X] Yes I've checked them.

### I have checked that the API documentation doesn't solve my issue.

- [X] Yes I've checked them.

### I have checked that the Box Developer Forums doesn't solve my issue.

- [X] Yes I've checked them.

### I have searched Issues in this repo and my issue isn't already reported.

- [X] Yes I've checked them.

### Describe the bug

When using folders command [`box folders:update`](https://github.com/box/boxcli/blob/main/docs/folders.md#box-foldersupdate-id) to update the [description](https://developer.box.com/reference/resources/folder/#param-description) field with the string that contains the backslash character (e.g. `Test\nTest\nTest`), the backslash character seems to be automatically escaped, and the request is sent with `Test\\nTest\\nTest`.

Because the backslash gets escaped, I'm unable to use `\n` to create new line characters. 

When running the command `box folders:update 47927266486 --description="Test\nTest\nTest"`, the description of the folder becomes 

> Test\nTest\nTest

although what I want is

> Test
> Test
> Test

### Expected behavior

The backslash character shouldn't have to be escaped so that the original string supplied by the user is respected.

When running the command `box folders:update 47927266486 --description="Test\nTest\nTest"`, the description of the folder should become 

> Test
> Test
> Test

instead of 
> Test\nTest\nTest

### Steps to reproduce

Run the command that contains "\n" within the parameter.

e.g. 

```
 box folders:update 47927266486 --description="Test\nTest\nTest"
```

### Authentication method used in your application

Server Authentication (with JWT)

### App Access Level

App + Enterprise Access

### What is Box CLI Version and Node used?

@box/cli/3.5.0 win32-x64 node-v14.19.3

### What is your Operating  System Version?

Windows

**Comment by antusus** [comment]
Hello @kojimaru14 ,
I can replicate this error. Thanks for all the details. I suspect this is `box-node-sdk` problem but I'll verify and we will work on a fix. Will let you know once we have this solved.

**Comment by arjankowski** [comment]
Hi @kojimaru14,
Thanks for reporting this issue!
We've just released a new version `v3.9.2` with the fix for this bug.

Best,
Artur
